Source/OxyPlot.SharpDX.WPF45/D3D11Image.cs

```diff
@@ -1,75 +1,90 @@
-﻿//Stolen from https://github.com/sharpdx/Toolkit/blob/master/Source/Toolkit/SharpDX.Toolkit.Game/Desktop/D3D11Image.cs
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="D3D11Image.cs" company="OxyPlot">
+//   Copyright (c) 2014 OxyPlot contributors
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
 
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using System.Windows.Interop;
-using SharpDX.Direct3D9;
-using SharpDX.Direct3D11;
-using DXGIResource = SharpDX.DXGI.Resource;
-using Device = SharpDX.Direct3D9.Device;
-using System.Windows;
+// Stolen from https://github.com/sharpdx/Toolkit/blob/master/Source/Toolkit/SharpDX.Toolkit.Game/Desktop/D3D11Image.cs
```

Comment:
I'm have a bit problems with copyrights. The original copyright is "Copyright (c) 2010-2014 SharpDX - Alexandre Mutel". So i changed header copyright tags to
<copyright file="D3D11Image.cs" company="SharpDX">
   Copyright (c) 2010-2014 SharpDX - Alexandre Mutel
</copyright>

But this gives me StyleCop Warning The company attribute in the file header must contain the following company name: OxyPlot	

and 

SA1636 : CSharp.Documentation : The copyright tag in the file header must contain the following text: Copyright (c) 2014 OxyPlot contributors.

How should i change header?